### ISSUE TITLE
"Mark room as Unread" with a 'dot' on the room list

### ISSUE THREAD

**Issue by daniellekirkwood** [vector-im/element-meta#891]
### Your use case

Users should be able to mark a room as unread from the context menu in the room list (long-press on mobile, 3 dots on web & desktop). 

In order to keep this deliverable simple we will: 
* Add a dot or other visual element to the room - there's no count or message specificity, read marker or read receipt movement. Just the introduction of a visual thing on the room list to jog the user's memory or remind them to 'come back later' 

We will _not_:
* Make this a cross-platform feature. Doing so would require engineers from each platform and some sort of user level storage which we may choose to do in the future but we want a quick and easy implementation that we can start with to judge the impact and usefulness of an unread marker. 

### Related platform issues
* The iOS issue for this: https://github.com/vector-im/element-ios/issues/7253

**Comment by daniellekirkwood** [comment]
Assigning to you Amsha as we'll need a design for how this should look visually. I imagine we could just add a grey dot to the room but we might want to change the colour or placement? Especially given we're also working on: https://github.com/vector-im/element-meta/issues/889

**Comment by erkinalp** [comment]
The thing is it would only mark as unread in your POV, not the public read marks.

**Comment by daniellekirkwood** [comment]
Yes, it's a tool to mark your own room list in a way that reminds you to come back to a room later

**Comment by phipz** [comment]
Why has it been decided to not make this syncing/cross-platform?

SchildiChat implements synced unread-markers, which even syncs unread markers with the WhatsApp bridge bot. As SchildiChat is an Element fork, it should be reasonable to adopt their implementation?

![CleanShot 2023-03-14 at 14 58 37](https://user-images.githubusercontent.com/29239149/225042765-a8ff80dd-7e4e-45e0-91d3-95f42b28ae88.gif)

(Right: WhatsApp Web, Left: SchildiChat)

**Comment by daniellekirkwood** [comment]
We'd love to make this cross-platform and have plans to do so in the future. Unfortunately other priorities have slowed us down in delivering that. 

We'd certainly be open to accepting the contribution of this feature if there were Element forks that wanted to provide that back to us :)

**Comment by phipz** [comment]
Here are the various open ends that might be helpful to push cross-platform implementation:

- Matrix Spec MSC2867 https://github.com/matrix-org/matrix-spec-proposals/pull/2867
- matrix-react-sdk implementation
  - Closed Pull request in Matrix.org https://github.com/matrix-org/matrix-react-sdk/pull/6314
  - Merged pull request in SchildiChat's fork https://github.com/SchildiChat/matrix-react-sdk/pull/2
- SchildiChat Desktop Implementation https://github.com/SchildiChat/schildichat-desktop/issues/109

**Comment by daniellekirkwood** [comment]
Adding more info to the title for clarity but am not defining the design (although we did land on a dot...)

**Comment by florian-bellencontre** [comment]
Since the feature is available on iOS, users don't understand why it's not available on other clients (Element web, desktop, android). This feature is one of the most highly awaited whenever I hear about Element.
